packages/nitro-protocol/test/contracts/ConsensusApp/validTransition.test.ts

```diff
@@ -27,15 +25,15 @@ beforeAll(async () => {
 
 describe('validTransition', () => {
   it.each`
-    isValid  | votesReqd | outcomes          | proposedOutcomes  | description
-    ${true}  | ${[0, 2]} | ${['0x1', '0x1']} | ${['0x1', '0x2']} | ${'valid consensus -> propose'}
-    ${false} | ${[0, 1]} | ${['0x1', '0x1']} | ${['0x1', '0x2']} | ${'invalid consensus -> propose: votesReqd too low'}
-    ${true}  | ${[2, 1]} | ${['0x1', '0x1']} | ${['0x2', '0x2']} | ${'valid vote'}
-    ${false} | ${[1, 1]} | ${['0x1', '0x1']} | ${['0x2', '0x2']} | ${'invalid vote: votesReqd not decreased'}
-    ${true}  | ${[1, 2]} | ${['0x1', '0x1']} | ${['0x2', '0x1']} | ${'valid veto'}
-    ${true}  | ${[2, 2]} | ${['0x1', '0x1']} | ${['0x2', '0x2']} | ${'valid pass'}
-    ${true}  | ${[1, 0]} | ${['0x1', '0x2']} | ${['0x2', '0x']}  | ${'valid finalVote'}
-    ${false} | ${[1, 0]} | ${['0x1', '0x3']} | ${['0x2', '0x']}  | ${'invalid finalVote: proposedOutcome1 ≠ currentOutcome2'}
+    isValid  | votesReqd | outcomes            | proposedOutcomes    | description
+    ${true}  | ${[0, 2]} | ${['0x01', '0x01']} | ${['0x01', '0x02']} | ${'valid consensus -> propose'}
```

Comment:
This change suggests that these should be
```suggestion
    ${true}  | ${[0, 2]} | ${[one, one]} | ${[one, two]} | ${'valid consensus -> propose'}
```

I don't think this PR is the right time to do that :P